src/test/java/org/folio/rest/impl/ApiTestBase.java

```diff
@@ -163,6 +163,27 @@ Response verifyGet(String url, Headers headers, String expectedContentType, int
     return verifyGet(url, APPLICATION_JSON, 200).as(clazz);
   }
 
+  Response verifyPostStartValueResponse(String url, String body, Headers headers, String
```

Comment:
This method does the same as `verifyPostResponse()`.

Reply:
Done, using the existing method.